src/providers/execution/contracts/staking_router.py

```diff
@@ -58,3 +58,30 @@ def get_all_node_operator_digests(self, module: StakingModule, block_identifier:
 
 class StakingRouterContractV2(StakingRouterContract):
     abi_path = './assets/StakingRouterV2.json'
+
+    @lru_cache(maxsize=1)
```

Comment:
not sure that it is good idea to have cache and `block_identifier = 'latest'` by default at the same time

Reply:
This is a special cache, that ignores 'latest' and default values and caches only hashes and numbers